src/server.h

```diff
@@ -1340,11 +1339,7 @@ struct redisServer {
                                      * value means fractions of microsecons (on average). */
     /* Pipe and data structures for child -> parent info sharing. */
     int child_info_pipe[2];         /* Pipe used to write the child_info_data. */
-    struct {
-        int process_type;           /* AOF or RDB child? */
-        size_t cow_size;            /* Copy on write size. */
-        unsigned long long magic;   /* Magic value to make sure data is valid. */
-    } child_info_data;
+    int child_info_nread;           /* last N bytes that parent read from pipe */
```

Comment:
maybe we can make the comment better describe the role of this variable.
something like this maybe?
```c
/* Num of bytes of a partial message read from the child info pipe. */
```